components/LoginPanel.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Button, Card, Form, Input, message } from 'antd';
+import { useRouter } from 'next/router';
+
+interface Props {
+  text: string;
+  isSignUp?: boolean;
+}
```

Comment:
I still don't know the difference between `type` and `interface`, but I been using `type` for most of my components. I think we should try and keep it consistent throughout the codebase. Also, could we change this to be `ComponentProps` for example:

```
type LoginPanelProps = {
   ...
}
```